src/lqcut.jl

```diff
@@ -0,0 +1,133 @@
+f_lq_DEP(x, v) = aoe_compton_signal_peakshape(x, v.μ, v.σ, v.n)
+
+
+"""
+    fit_lq_DEP_ampl(h::Histogram, ps::NamedTuple{(:peak_pos, :peak_fwhm, :peak_sigma, :peak_counts, :mean_background, :μ, :σ), NTuple{7, T}}; uncertainty::Bool=true) where T<:Real
+
+Perform a fit of the peakshape to the data in `h` using the initial values in `ps` while using the `f_lq_DEP` function consisting of a gaussian peak.
+
+# Returns
+    * `result`: NamedTuple of the fit results containing values and errors
+    * `report`: NamedTuple of the fit report which can be plotted
+"""
+function fit_lq_DEP(h::Histogram, ps::NamedTuple; uncertainty::Bool=true)
```

Comment:
Move the actual `Normal` fit maybe into another function. Best would be to include it in the `fit_single_trunc_gauss` function, to be able to also fit binned data. 
This would make the code more clear and readable.